test/testapps/TestServer/Controllers/PersonController.cs

```diff
@@ -32,6 +33,23 @@ public IEnumerable<string> Get()
             }
         }
 
+        [HttpPost("xhrf")]
```

Comment:
Are these methods `GetAntiForgery` and `PostAntiForgery` used anywhere? Could you clarify their purpose?

Reply:
I added these methods to test antiforgery cookies submition, the e2e test is missing.  I'm going to add it